modules/openapi-generator/src/main/resources/python-nextgen/api.mustache

```diff
@@ -62,10 +63,6 @@ class {{classname}}(object):
 {{/allParams}}
         :param async_req: Whether to execute the request asynchronously.
         :type async_req: bool, optional
-        :param _preload_content: if False, the urllib3.HTTPResponse object will
```

Comment:
It's very useful switch, especially if you need data in a raw form for some reason (performance mainly). Can we keep this in some form?

Reply:
Why not store the http raw response in ApiRsponse?

Reply:
Sure, ApiResponse.raw_data is the best place, but if _preload_content is False, deserialize() is not called and ApiResponse.data is None.

Reply:
That's fine. One could keep raw_data as optional and add a comment describing that

Reply:
@tomplus  The data in raw form (HTTP response body without deserialisation) is stored in `raw_data` in ApiResponse. Would that meet your requirement?

Reply:
UPDATE: added back the switch

Reply:
Looks good to me, thanks!